notebooks/01_introduction.ipynb

```diff
@@ -301,7 +301,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.4"
+   "version": "3.12.0"
```

Comment:
@Azaya89 Would it also make sense to update environment.yml as well while you are at it? It currently states `python>=3.10` (and `bokeh>=3.2.0`)!

Reply:
I can update the bokeh to `>=3.3.1` but is it necessary to update the python beyond `3.10`?

Reply:
Not strictly necessary, but since this repo is only updated sporadically, we generally try to do general updates alongside whenever those updates happen. E.g. new update to latest python version pro-actively so that we can hopefully keep ahead of general Bokeh `minpy` advances. Do you have a specific concern with updating the env file?

Reply:
Oh, I do not have any specific concerns actually. I was just curios about the reason. I guess it makes sense to update it to `3.12` then...